retry.go

```diff
@@ -72,8 +72,6 @@ func (c Client) newRetryTimer(ctx context.Context, maxRetry int, unit time.Durat
 		for i := 0; i < maxRetry; i++ {
 			select {
 			case attemptCh <- i + 1:
-			case <-ctx.Done():
-				return
```

Comment:
Needs to remove it because FPutObjectWithContext should fail on short timeout, and this case hold it and avoid to receive error.